### ISSUE TITLE
[False-Positive] Variable "assign_delegate_screen" not used

### ISSUE THREAD

**Issue by lucasborin** [abaplint/abaplint#1902]
![image](https://user-images.githubusercontent.com/5233413/113911390-4e61c280-97b0-11eb-85ee-fca4cdeb5b5e.png)

![image](https://user-images.githubusercontent.com/5233413/113911432-591c5780-97b0-11eb-9ab5-4d3c2212eb02.png)

Run: https://github.com/SAP/code-pal-for-abap/pull/304/checks?check_run_id=2289709342

**Comment by larshp** [comment]
reproduced as

```abap
DATA bar TYPE c LENGTH 1 VALUE 'A'.
SET PF-STATUS bar.

DATA bar TYPE c LENGTH 1 VALUE 'A'.
SET TITLEBAR bar.
```

**Comment by lucasborin** [comment]
Issue fixed. 

Run: https://github.com/SAP/code-pal-for-abap/pull/304/checks?check_run_id=2296641852
